lux/lib/lux/prisms/goat_sdk/uniswap_swap.ex

```diff
@@ -27,6 +27,11 @@ defmodule Lux.Prisms.GoatSdk.UniswapSwap do
           type: :integer,
           description: "Slippage tolerance in basis points (e.g. 50 for 0.5%)",
           default: 50
+        },
+        api_key: %{
```

Comment:
We shouldn't pass API keys as args. We should load them from config. Please take a look at `config/runtime.exs` and `Lux.Config` to see how we expose Configs to the whole app.

I am thinking now that we don't have a native way of doing it for python code (I actually don't know what's the usual way to do it^^)